Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBarItemTemplateSelector.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// <see cref="DataTemplateSelector"/> used by <see cref="TabbedCommandBar"/> for determining the style of normal vs. contextual <see cref="TabbedCommandBarItem"/>s.
+    /// </summary>
+    public class TabbedCommandBarItemTemplateSelector : DataTemplateSelector
```

Comment:
Did you investigate the style selector vs. data template?

Reply:
I've been trying to figure out how to override those NavigationViewItem resources in a style. I think it has to be a template for it to work

Reply:
Ah, I see what you mean around the overridden resources... good point. Hmm. I'll think about that, for now we can leave it.